src/vcbuild/dmd_backend.vcxproj.filters

```diff
@@ -1,446 +1,443 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This is probably converting CRLF to LF. This should not happen for the VC projects.

Reply:
Changed it back to CRLF.